### ISSUE TITLE
Downloading configuration dependency sources/javadoc in Gradle project causes Configurations project section to disappear

### ISSUE THREAD

**Issue by gwinstanley** [apache/netbeans#6059]
### Apache NetBeans version

Apache NetBeans 18

### What happened

To get source/javadoc for a dependency in a Gradle project, I often find the dependency in the Configurations/compileClasspath section of the project, then right-click to download the relevant remote JAR from the repo. Doing this seems to succeed, but the entire Configurations section of the project disappears, and a project reload seems to be required to make it reappear (with the download having succeeded).

### How to reproduce

1. Create/open a Gradle project
2. Ensure it has at least one dependency
3. Ensure project is compilable
4. Use context menu on dependency in Configurations/compileClasspath (or other config) to download sources/javadoc
5. Observe result

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

macOS 13.4 (Intel)

### JDK

Temurin OpenJDK 20.0.1+9 (x86_64)

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else

Seems to happen every time.
I recall it also happening on prior NB versions, so it's a long-standing issue, I believe.

### Are you willing to submit a pull request?

No

**Comment by gwinstanley** [comment]
**Update:** on NB27 this behaviour seems to have changed; instead of the project section disappearing, the (successfully) downloaded sources/javadoc is not registered/recognized by the IDE, so it continues to display in the tree as though it still requires downloading. However, looking in the Gradle repo cache the downloaded artifacts have indeed been downloaded.

**TLDR:** this seems to be a lesser issue now of simply needing the IDE's knowledge of successful download refreshed.
